pyproject.toml

```diff
@@ -42,6 +42,7 @@ inquirerpy = "^0.3.4"
 google-cloud-storage = "2.10.0"
 loguru = ">=0.7.2"
 uvloop = "^0.19.0"
+halo = "^0.0.31"
```

Comment:
@anesmemisevic  I think you will need to also add this to the builder requirements group as well so integration tests pass.

Reply:
@bolasim we may also need it in group.dev, correct?

Reply:
dev doesn't need it since we assume we installed all these default deps.